packages/@aws-cdk/aws-route53/README.md

```diff
@@ -123,4 +123,51 @@ you know the ID and the retrieval for the `zoneName` is undesirable.
 const zone = HostedZone.fromHostedZoneId(this, 'MyZone', {
   hostedZoneId: 'ZOJJZC49E0EPZ',
 });
+```
+
+
+## VPC Endpoint Service Private DNS
+
+When you create a VPC endpoint service, AWS generates endpoint-specific DNS hostnames that consumers use to communicate with the service.
+For example, vpce-1234-abcdev-us-east-1.vpce-svc-123345.us-east-1.vpce.amazonaws.com.
+By default, your consumers access the service with that DNS name.
+This can cause problems with HTTPS traffic because the DNS will not match the backend certificate:
+```console
+curl: (60) SSL: no alternative certificate subject name matches target host name 'vpce-abcdefghijklmnopq-rstuvwx.vpce-svc-abcdefghijklmnopq.us-east-1.vpce.amazonaws.com'
+```
+Effectively, the endpoint appears untrustworthy. To mitigate this, clients have to create an alias for this DNS name in Route53.
+
+Private DNS for an endpoint service lets you configure a private DNS name so consumers can
+access the service using an existing DNS name without creating this Route53 DNS alias
```

Comment:
It kind of *is* a R53 alias, as if AWS was managing the hosted zone and ARecord for a service client.

For services exposed over a PrivateLink, a client has to set up the alias themselves -- create the hosted zone, ARecord, etc, because the service has no idea what the VPC endpoint DNS name is going to be (autogenerated on creation).

Private DNS basically means AWS will manage the DNS for the client, so the hostname will resolve correctly to the VPC endpoint without the client having to create a hosted zone or an ARecord.